index.bs

```diff
@@ -810,6 +810,52 @@ You may follow the guidelines for <a href="bfcache">BFCache</a> mentioned above,
 as we expect BFCached and detached documents to be treated the same way,
 with the only difference being that BFCached documents can become [=Document/fully active=] again.
 
+<h3 class=question id="accessibility-devices">
+  Does your feature allow sites to learn about the users use of accessibility devices?
+</h3>
+The Web is designed to work for everyone, and Web standards should be designed
+for people using assistive devices just as much as for users relying
+on mice, keyboards, and touch screens. Accessibility and universal access
+are core to the W3C's mission.
+
+Standards authors though should keep in mind that Web users that rely on
+assistive devices and technologies face some unique risks when using the Web.
+The use of assistive technologies may cause those Web users to stand
+out among other Web users, increasing the risk of unwanted reidentification
+and privacy harm. Similarly, some Web site operators may try to
+discriminate against Web users who rely on assistive devices.
+
+Feature designers and standards authors should therefor be thoughtful and
+careful to limit if, and what, websites can learn about the use off assistive
+technologies. Standards authors must minimize both what inflation about
+accessibility device use their features reveal, both explicitly
+and implicitly. Examples of <em>explicit</em> information about assistive devices
+include device identifiers, model names. Examples of <em>implicit</em>
+information about the use of assistive devices might include
+user interaction patterns that are unlikely to be generated by a
+mouse, keyboard, or touch screen.
+
+<p class=example>
+The [[wai-aria-1.3]] defines additional markup site authors can use to make
+their pages easier to navigate with accessibility devices. The standard
+includes the an [`aria-hidden`](https://w3c.github.io/aria/#aria-hidden)
+attribute, that site authors can use to indicate that certain content
+should be hidden from assistive devices.
+
+A malicious site author might
+abuse the `aria-hidden` attribute to learn if a user is using assistive
+technology, possibly by revealing certain page content to assistive devices,
+while showing very different page content to other users. A malicious
```

Comment:
Similar things are true for CSS feature detection for reduced motion or high contrast.  Worth noting?

Reply:
@martinthomson thats a good example, but i'd like to keep the document lean if possible, and so avoid duplicate examples unless needed.  But if you disagree or think we need more examples to demonstrate other edges of the problem, lets follow up in a new issue